pkg/virt-launcher/virtwrap/network/podinterface.go

```diff
@@ -785,6 +806,10 @@ func (p *MasqueradePodInterface) preparePodNetworkInterfaces(queueNumber uint32,
 		} else {
 			return fmt.Errorf("Couldn't configure ipv6 nat rules")
 		}
+
+		if err := Handler.CreateNDPConnection(p.bridgeInterfaceName, launcherPID); err != nil {
+			return fmt.Errorf("could not start the RA daemon: %v", err)
```

Comment:
1. The error massage is not accurate. A daemon is not started, only a connection is created.
2. In case of an error we should close the opened fd.

Reply:
yes